### DISCUSSION TITLE
Matrix view - quick editing and/or drag/drop

### DISCUSSION THREAD

**Discussion by Wassa123** [Zuehlke/poinz#249] [Ideas]
I'm loving the estimation and the matrix view.

I'm trying to set up baselines with my team, where the matrix view is a bit too fixed.
There are cases where we'd like to adjust a previously estimated story, but we have to click out of the view, click the story, and edit the points, and click back.

Is it possible to add some functionality to quick edit a story on there, maybe just allowing the dragging to another column?

**Comment by xeronimus** [comment]
Hi there,

thank you very much for the suggestion.

Let's start finding out whether we should implement this. Technically it's not a problem.

Some thoughts from my side :

* Setting the estimation value (consensus) "manually" - possibly for multiple stories - on a room-wide story overview is quite a bit "apart" from the main use case of having a "Poker Estimation Round"  (everybody estimates together on one story at a time)
What are the consequences of providing this feature?

* Currently you can only settle the estimation value on the "currently active story" and only if that active story is revealed. [Because the flow was:  select a story, start estimating as a team, discussing, re-estimate, settle if necessary]
   should we remove these constraints, so that you could settle on any story at any time?

* what happens to the other users that are currently not seeing the matrix, but maybe are simultaneously estimating a single story (pokering) - and somebody in the room manipulates stories via the matrix?

**Reply by Wassa123** [reply]
Hello,

1) We had just started using points, and have not yet established a baseline, and so we were not sure how many points each item should be.
So we started estimating points, and then as more stories came in, that fitted between existing stories, we found that we had to move previously estimated ones about. However we couldn't do this, and then had to go through each ticket manually and change the points.

2) We can just select an existing story and press 'estimate' which goes back to that story anyway.

3) I don't think that would happen, usually when we want to do this, we're talking about it over a screen share session, and so we wouldn't be doing both at the same time.

As a side thought, is it possible to specify "admins" that have access to do these things?

**Reply by xeronimus** [reply]
2. I think we will adjust the preconditions as follows:  a story can be "settled" (also via the matrix) as long at is has a consensus value. we do no longer need to "select" it first.
(a story must have a consensus to appear on the matrix (and thus be revealed) anyway.)
3. keep in mind, that different teams use PoinZ in various ways. however, if we do not "activate" a story when moving it on the matrix (see 2.)  then it should work nicely.

**Reply by xeronimus** [reply]
> Issue https://github.com/Zuehlke/poinz/issues/253
